src/test/java/one/util/streamex/TestHelpers.java

```diff
@@ -20,17 +20,16 @@
 import java.util.*;
 import java.util.Map.Entry;
 import java.util.concurrent.ConcurrentLinkedQueue;
+import java.util.concurrent.Semaphore;

```

Comment:
Is this import still necessary?

Reply:
Oops. `import static java.util.concurrent.TimeUnit.SECONDS;` is also outdated.